BinaryEncoding.md

```diff
@@ -786,9 +783,140 @@ for [future :unicorn:][future multiple tables] use and must be 0 in the MVP.
+| `i32.atomic.rmw16_s.and` | `0xe614` | `2` `memory immediate` | atomic and with memory |
+| `i32.atomic.rmw16_u.and` | `0xe615` | `2` `memory immediate` | atomic and with memory |
+| `i64.atomic.rmw8_s.and` | `0xe616` | `2` `memory immediate` | atomic and with memory |
+| `i64.atomic.rmw8_u.and` | `0xe617` | `2` `memory immediate` | atomic and with memory |
+| `i64.atomic.rmw16_s.and` | `0xe618` | `2` `memory immediate` | atomic and with memory |
+| `i64.atomic.rmw16_u.and` | `0xe619` | `2` `memory immediate` | atomic and with memory |
+| `i64.atomic.rmw32_s.and` | `0xe61a` | `2` `memory immediate` | atomic and with memory |
+| `i64.atomic.rmw32_u.and` | `0xe61b` | `2` `memory immediate` | atomic and with memory |
+| `i32.atomic.rmw.or` | `0xe610` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic.rmw.or` | `0xe611` | `3` `memory immediate` | atomic or with memory |
+| `i32.atomic.rmw8_s.or` | `0xe612` | `3` `memory immediate` | atomic or with memory |
+| `i32.atomic.rmw8_u.or` | `0xe613` | `3` `memory immediate` | atomic or with memory |
+| `i32.atomic.rmw16_s.or` | `0xe614` | `3` `memory immediate` | atomic or with memory |
+| `i32.atomic.rmw16_u.or` | `0xe615` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic.rmw8_s.or` | `0xe616` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic.rmw8_u.or` | `0xe617` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic.rmw16_s.or` | `0xe618` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic.rmw16_u.or` | `0xe619` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic.rmw32_s.or` | `0xe61a` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic.rmw32_u.or` | `0xe61b` | `3` `memory immediate` | atomic or with memory |
+| `i32.atomic.rmw.xor` | `0xe610` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic.rmw.xor` | `0xe611` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic.rmw8_s.xor` | `0xe612` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic.rmw8_u.xor` | `0xe613` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic.rmw16_s.xor` | `0xe614` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic.rmw16_u.xor` | `0xe615` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic.rmw8_s.xor` | `0xe616` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic.rmw8_u.xor` | `0xe617` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic.rmw16_s.xor` | `0xe618` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic.rmw16_u.xor` | `0xe619` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic.rmw32_s.xor` | `0xe61a` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic.rmw32_u.xor` | `0xe61b` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic.rmw.xchg` | `0xe610` | `5` `memory immediate` | atomic exchange with memory |
```

Comment:
Is xchg actually a RMW operation? Seems like it could be its own group, with names like `i32.atomic.xchg16_s`, in line with `cmpxchg`.